src/View/Helper/CdnHelper.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace BootstrapUI\View\Helper;
+
+/**
+ * CndHelper class to insert Bootstrap CDN css and javascript assests into layout.
+ *
+ */
+ class CdnHelper extends \Cake\View\Helper
+{
+
+    /**
+     * @var xxx
+     */
+    public $helpers = [
+        'Html'
+    ];
+
+    /**
+     * Default config
+     *
+     * - css: Full URL to most current TwitterBootstrap minified css
+     * - script: Full URL to most current TwitterBootstrap minified js
+     *
+     * @var array
+     */
+    protected $_defaultConfig = [
+        'css' => 'http://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css',
+        'script' => 'http://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js'
```

Comment:
I understood we should stay away from Configure options to keep things as transparent as possible but functional users can already specify their own versions in AppView if these default two don't suit them:

```php
    public function initialize()
    {
        $this->loadHelper('Cdn', [
            'className' => 'BootstrapUI.Cdn',
                'css' => 'http://my.version.',
                'script' => 'http://my.version'
        ]);
    }
```

If I am missing something please let me know.